packages/babel-helper-create-class-features-plugin/src/misc.js

```diff
@@ -98,10 +98,14 @@ export function extractComputedKeys(ref, path, computedPaths, file) {
       const ident = path.scope.generateUidIdentifierBasedOnNode(
         computedNode.key,
       );
+      path.scope.push({
+        id: ident,
+        kind: "let",
```

Comment:
I don't see any advantage in using `let` here? Usually we try to generate an output as low-level as possible, if it isn't harder to do.

Reply:
It is for this scenario: https://github.com/Microsoft/TypeScript/issues/27864

if it is set as `var`, it will behave the same as pointed out in the issue link above

maybe it should be determined by whether the computedKeys original declaration kind, to preserve the original scoping intention?

Reply:
Oh ok; it's ok like this then.

Reply:
Seems like adding a comment explaining this might be useful?